.github/compatibility-test-matrices/main/client-chain-a.json

```diff
@@ -20,6 +20,7 @@
   ],
   "test": [
     "TestClientUpdateProposal_Succeeds",
+    "TestClient_Update_Misbehaviour",
```

Comment:
Noticed this was missing, so I added it for free in this PR.

Reply:
I feel like we should drop an underscore `TestClientUpdate_Misbehaviour` 😅 but its fine!

Reply:
I'd be happy with that change too :D, would need to make sure to update all the other json files though